src/engines/fx/chorus/screen.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+
+#include "core/ui/vector_graphics.hpp"
+#include <Gamma/Envelope.h>
+#include "chorus.hpp"
+
+
+
+namespace otto::engines::chorus {
+
+  using namespace core;
+
+  struct Screen : ui::Screen {
+    void draw(nvg::Canvas& ctx) override;
+    void draw_front_head(nvg::Canvas&, nvg::Point, ui::vg::Color, float);
+    void draw_background_head(nvg::Canvas&, nvg::Point, ui::vg::Color, float);
+
+    float delay_ = 0.f;
```

Comment:
all of these variables should be private and declared down by the other variables.